models/mesh.py

```diff
@@ -49,20 +50,20 @@ def __str__(self):
     def __repr__(self):
         return "{}: {}".format(self.id, self.__str__())
 
-@event.listens_for(Mesh, "after_insert")
-@event.listens_for(Mesh, "after_update")
-def receive_after_update(mapper, connection, target):
-    """
-    event listener, to be executed when a zipfile is provided for upload
-    """
-    src_path = "mesh"
-    trg_path = f"static/{src_path}/{target.id}"
-    if target.zipfile is not None:
-        zipfile = os.path.join(src_path, target.zipfile)
-        if not(os.path.isfile(zipfile)):
-            raise IOError(f"File {zipfile} does not exist")
-        unzipmesh(zipfile, trg_path=trg_path, mesh_name="unity")
-        # no processing needed, so set Mesh status to finished immediately
-        print("Set Mesh status to 'Finished'")
-        target.status = 3
-
+# @event.listens_for(Mesh, "after_insert")
+# @event.listens_for(Mesh, "after_update")
+# def receive_after_update(mapper, connection, target):
+#     """
+#     event listener, to be executed when a zipfile is provided for upload
```

Comment:
Seems obvious that it's an event listener; wouldn't this comment be more helpful if it specified what it does? I.e. ```# When zipfile containing a mesh is provided for upload, unzips it and places it where the mesh is used by the game engine.```

Reply:
I had to read code to figure out what was going on, which could have been explicit in the comment!